src/main/java/io/github/apace100/apoli/util/InventoryUtil.java

```diff
@@ -386,13 +389,25 @@ public static void forEachStack(Entity entity, Consumer<ItemStack> itemStackCons
     }
 
     private static void deduplicateSlots(Entity entity, Set<Integer> slots) {
```

Comment:
Are we sure that this method should be kept here? We aren't using it anymore and it also isn't exposed for anybody else to use, so I feel like we could remove it.

Reply:
There's two things we can do here as this method **is** used.

1. Ignore this review.
2. Implement the optimisations to the set creation inside the other `InventoryUtil` methods.